integration-tests/rpc-nodes.spec.ts

```diff
@@ -409,13 +409,13 @@ CONFIGS().forEach(
           done();
         });
 
-        jakartanet('Verify that rpcClient.getTxRollupInbox will access the inbox of a transaction rollup on jakartanet', async (done) => {
+        limanet('Verify that rpcClient.getTxRollupInbox will access the inbox of a transaction rollup on jakartanet', async (done) => {
```

Comment:
Should this test run on all testnets? Using `it` rather than `limanet`?
I am not too sure why they were not running on Kathmandu..